.github/workflows/run-tests.yml

```diff
@@ -1,109 +1,53 @@
-# Implicit requirements
-# runner must have `docker` and `curl` installed (true on github-runners)
-
 name: run-tests
 
 on:
-  workflow_call:
-  workflow_dispatch:
```

Comment:
I want to have ability to manual run workflow. Event type `workflow_dispatch` should be enabled for that
https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/manually-running-a-workflow